index.js

```diff
@@ -33,13 +33,13 @@ module.exports = (filename, opts) => {
 
 		if (file.isNull() && file.stat && file.stat.isDirectory && file.stat.isDirectory()) {
 			zip.addEmptyDirectory(pathname, {
-				mtime: file.stat.mtime || new Date(),
+				mtime: opts.mtime || file.stat.mtime || new Date(),
 				mode: file.stat.mode
 			});
 		} else {
 			const stat = {
 				compress: opts.compress,
-				mtime: file.stat ? file.stat.mtime : new Date(),
+				mtime: opts.mtime || (file.stat ? file.stat.mtime : new Date()),
```

Comment:
I would name the option `modifiedTime`. It's more readable.

Reply:
I totally agree, and that was my first reaction. But then I changed my mind and went with the same name that YAZL uses because, after all, this change is simply surfacing a YAZL option. I'll go with whichever way you prefer, but let me know how you want to proceed with that in mind.

Reply:
I forgot to mention that, not just YAZL, but Node.js itself also uses `mtime` for its `fs.stat()`-related APIs. Regardless, I'm changing it to `modifiedTime`; even though `mtime` is the precedent, that doesn't mean it's a _good_ precedent.